src/format/parse.rs

```diff
@@ -424,27 +451,24 @@ where
                     | &TimezoneOffsetDoubleColon
                     | &TimezoneOffsetTripleColon
                     | &TimezoneOffset => {
-                        let offset = try_consume!(scan::timezone_offset(
-                            s.trim_left(),
-                            scan::colon_or_space
-                        ));
+                        s = scan::space1(s);
```

Comment:
why do we allow one optional whitespace here?

Reply:
Previously, infinite whitespace was allowed. So strftime specifier `%Y/%m/%d %H:%M:%S%z` would match values `"2022/01/31 00:01:59-0700"`, and  `"2022/01/31 00:01:59 -0700"`, and `"2022/01/31 00:01:59          \n\t\n -0700"`.

I guessed a significant portion of chrono users would be affected. So I erred away from too many breaking changes.
Allowing one whitespace seemed like a reasonable compromise , e.g. `"2022/01/31 00:01:59-0700"` and `"2022/01/31 00:01:59 -0700"` will still match the specifier string.

If you would prefer to be strict about this, e.g. only `"2022/01/31 00:01:59-0700"` would match the specifier string, then that would be fine with me. My personal preference is to be strict because that simplifies reasoning about correctness.

Reply:
I think that given this will be a breaking change anyway, lets be explicit about all white space rather than being relaxed in some areas and explicit in others. 

Even in the case that we do decide to leave this optional white space in, we could probably leave a comment justifying why